src/FX.Tests/E2ETests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+using FsharpExchangeDotNetStandard;
+
+using NUnit.Framework;
+using StackExchange.Redis;
```

Comment:
@webwarrior-ws to respect our style guidelines please separate the `using` statements above in 3 groups

Reply:
done